packages/atlas/src/components/NftTile/NftTileDetails.tsx

```diff
@@ -35,22 +36,24 @@ export type NftTileDetailsProps = {
   owner?: Member
   creator?: Member
   role?: 'owner' | 'viewer'
-  auction?: 'none' | 'minBid' | 'topBid' | 'waiting'
+  nftStatus?: 'idle' | 'on-sale' | 'auction'
   buyNowPrice?: number | null
-  minBid?: number | null
+  startingPrice?: number | null
   topBid?: number | null
   title?: string | null
   hovered?: boolean
+  interactable?: boolean
+  videoHref?: string
 }
 
 type TileSize = 'small' | 'medium'
 
-type DetailsContent = {
+type DetailsContentProps = {
```

Comment:
This is just a preference, but could you move this closer to the DetailContent component?